README.md

```diff
@@ -242,6 +248,18 @@ the relevant queries and mutations on demand.
 
 ## How to deploy a new instance ##
 
+Creating a instance of the bot requires to create a [GitHub App](https://docs.github.com/en/developers/apps/creating-a-github-app) and set up the server. 
+
+### Create GitHub app
+
+[repository permissions](screenshots/github_app_repository_permissions.png)
+
+[organization permissions](screenshots/github_app_organization_permissions.png)
+
+[event subscriptions](screenshots/github_app_event_subscriptions.png)
```

Comment:
I think I would prefer a short checklist of which permissions need to be changed from the default (because it is easy to miss something in these long screenshots).